recipes/itk/bld.bat

```diff
@@ -0,0 +1,2 @@
+%PYTHON% -m pip install --no-deps https://pypi.org/packages/cp35/t/tensorflow/tensorflow-%PKG_VERSION%-cp35-cp35m-win_amd64.whl
```

Comment:
@thewtex I think you need to do the same thing here that we did for build.sh. But I have no idea how to do loops in windows batch scripts 🤷‍♂️